### ISSUE TITLE
Remove current Celestia DALC

### ISSUE THREAD

**Issue by tzdybal** [celestiaorg/optimint#155]

**Comment by adlerjohn** [comment]
Ideally this transaction generation and signing logic should be independent of Optimint, no? There's the possibility for rollups to not use Optimint and implement their own aggregator.

**Comment by tzdybal** [comment]
Yeah, we could move it into Optimint, but as @adlerjohn pointed out, it's a piece of code that should be as standalone as possible, so everyone can reuse it. This will be valuable for rollup aggregators and any other app/dapp/younameit willing to submit data to Celestia.
